z-client-mu-plugins.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+// Note: This file is prefixed with `z-` for load order
+
+define( 'WPCOM_VIP_CLIENT_MU_PLUGIN_DIR', WP_CONTENT_DIR . '/client-mu-plugins' );
+
+function wpcom_vip_load_client_mu_plugins() {
+	static $loaded = false;
+
+	// Prevent running this multiple times
+	if ( $loaded ) {
+		return;
+	}
+
+	$loaded = true;
+
+	// Code below is adapted from wp_get_mu_plugins
+	$client_mu_plugins = [];
+
+	if ( ! is_dir( WPCOM_VIP_CLIENT_MU_PLUGIN_DIR ) ) {
+		return;
+	}
+
+	$dh = opendir( WPCOM_VIP_CLIENT_MU_PLUGIN_DIR );
+	if ( ! $dh ) {
+		return;
+	}
+
+	do {
+		$plugin = readdir( $dh );
+		if ( false === $plugin ) {
+			break;
+		}
+
+		if ( substr( $plugin, -4 ) === '.php' ) {
+			$client_mu_plugins[] = WPCOM_VIP_CLIENT_MU_PLUGIN_DIR . '/' . $plugin;
+		}
+	} while ( false !== $plugin );
+
+	closedir( $dh );
+	sort( $client_mu_plugins );
```

Comment:
Presumably this is so files load in a predictable, alphabetical order instead of whatever order they happen to appear on the filesystem?